README.md

```diff
@@ -12,6 +12,10 @@ Here is my responsive portfolio
 - Html and CSS
 - VS studio as IDE
 
+## Live Demo
+
+[Live Demo Link](https://m05e5.github.io/moses_portfolio)
+
 
 ### Prerequisites
 
```

Comment:
- [ ] A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). 
- Focus on the instructions you gave for cloning the repo and running it locally.
- The screenshot you added in your readme is broken. Kindly fix it.

Reply:
Ok thanks for the review I will apply the changes but you didn't mention that in the first review

Reply:
Hi @m05e5 , I didn't mention that in the first review because it was invalidated. 😇